web/cobrands/sass/_base.scss

```diff
@@ -2244,6 +2244,16 @@ div.assigned-to {
     margin-top: 0.25em;
 }
 
+div.land-type-public {
```

Comment:
These should go into Peterborough's `base.scss` as it only affects that cobrand.